code/game/objects/items/weapons/material/material_weapons.dm

```diff
@@ -2,7 +2,7 @@
 // This class of weapons takes force and appearance data from a material datum.
 // They are also fragile based on material data and many can break/smash apart.
 /obj/item/material
-	hitsound = 'sounds/weapons/bladeslice.ogg'
+	hitsound = SFX_WOUND_SPLATTER
```

Comment:
Can't compare sounds right now, but are you sure this SFX is more appropriate than bladeslice.ogg? From the definition name, it sounds like it'd be better for a, well, splatter effect.

Reply:
Listened to the sound and it seemed appropriate, ALTHOUGH you should really move this to the types that actually are knives. Currently this hitsound will be used for shit like ashtrays.

In fact, this should probably be set as an abstract type. Slap a `base_type = /obj/item/material` on there for me please 🥺

Reply:
> Listened to the sound and it seemed appropriate, ALTHOUGH you should really move this to the types that actually are knives. Currently this hitsound will be used for shit like ashtrays.
> 
> In fact, this should probably be set as an abstract type. Slap a `base_type = /obj/item/material` on there for me please 🥺

base_type is undefined

Reply:
> > Listened to the sound and it seemed appropriate, ALTHOUGH you should really move this to the types that actually are knives. Currently this hitsound will be used for shit like ashtrays.
> > In fact, this should probably be set as an abstract type. Slap a `base_type = /obj/item/material` on there for me please 🥺
> 
> base_type is undefined

sorry, `abstract_type`

Reply:
ok done